README.MD

```diff
@@ -1 +1,18 @@
-Genome Down Sampling Wokflow
+<h2><b>Genome Downsampling Wokflow</b></h2>
+
+The genome downsampling workflow consists of <b>downsampling_part1</b> and <b>downsampling_part2</b> and executed from <b>Downsample.wdl</b>. The workflow utilizes <a href="https://github.com/samtools/samtools">Samtools</a>, <a href="https://github.com/broadinstitute/picard">Picard</a>, <a href="https://sourceforge.net/projects/bio-bwa/files/">BWA</a>, <a href="https://github.com/broadinstitute/gatk/releases/tag/4.1.9.0">GATK</a>, and <a href="https://github.com/dcjones/fastq-tools">fastq-tools</a> to randomly down-sample to a desired sequencing coverage.
+
+<b>Downsampling_part1:</b>
+Converts a CRAM/BAM input file into paired-end FASTQ files and extracts read groups from input file.
+
+<b>Downsampling_part2:</b>
+Handles downsampling to a specified final coverage and assumes an initial 30x coverage. It outputs read counts, downsampled CRAM file, coverage estimate, marked duplicate reads.
+
+<b><i>Note:</i></b> 
+Dockerfile installs: 
+<ul>
+<li>bwa version 0.7.17, which is different than publically available bwa internal index files if using bwakit (0.7.12).</li>
```

Comment:
Did you end up changing the docker version to match the public reference files, or are you using your own reference files?

Reply:
I tried using the public data but it didn't work, Harrison says it's not a bad idea to use a newer version file so for now I am sticking to my own references